lib/gui/app.js

```diff
@@ -100,9 +101,12 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
     currentVersion: packageJSON.version
   });
 
-  UpdateNotifierService.isLatestVersion().then((isLatestVersion) => {
+  const CURRENT_VERSION = ManifestBindService.get('version');
 
-    if (isLatestVersion) {
+  s3Packages.getLatestVersion(s3Packages.BUCKET_URL_PRODUCTION, {
+    currentVersion: CURRENT_VERSION
+  }).then((latestVersion) => {
+    if (latestVersion === CURRENT_VERSION) {
```

Comment:
I ended up returning `undefined` and converting to `0.0.0` if that's the case in the comparison place since returning `0.0.0` from this function might give the impression that there is really a version called `0.0.0` uploaded in the bucket.